src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -382,6 +395,7 @@ public async Task<IActionResult> SetDeploymentTarget(string sessionId, [FromBody
                     // This is because pushing images to ECR does not need a cloud application name.
                     if (!cloudApplicationNameGenerator.IsValidName(newDeploymentName))
                     {
+                        //throw new InvalidParameterException(cloudApplicationNameGenerator.InvalidNameMessage(newDeploymentName));
```

Comment:
This is commented out

Reply:
And if removing this comment, it looks like it would have been the only use of `InvalidParameterException` above so could we remove that too?

Reply:
Whoops. I was playing around with having custom exceptions but since you can only have a custom exception per Http status code it didn't seem like it was buying much. Removed the orphaned code.